thoth/storages/graph/postgres.py

```diff
@@ -3771,6 +3771,52 @@ def create_python_cve_record(
 
                 return False
 
+    def update_missing_flag_package_version(
+        value: bool,
+        index_url: str,
+        package_name: str,
+        package_version: str
+    ) -> None:
+        """Update value of is_missing flag for PythonPackageVersion."""
+        with self._session_scope():
+            (session.query(PythonPackageVersion)
+                .update()
+                .where(index_url == index_url)
+                .where(package_name == package_name)
+                .where(package_version == package_version)
+                .values(is_missing=value))
+
+    def get_repositories_using_package(index_url: str, package_name: str) -> List[str]:
```

Comment:
According to naming conventions for queries, this should have `_all` at the end. Moreover, in the docstring I would add `unique` urls of repos... becuase the `distinct` filter is not optional in this case. Same for `get_repositories_using_package_version`.